app/src/main/java/com/wizeline/academy/animations/core/data/repository/HomeRepository.kt

```diff
@@ -1,5 +1,7 @@
 package com.wizeline.academy.animations.core.data.repository
 
+import com.wizeline.academy.animations.ui.home.HomeItem
+import com.wizeline.academy.animations.ui.home.homeItem
```

Comment:
why double import?

Reply:
I don't know why it's 2 times but I need it because if not I can't use my extension function that is called `homeItem()`